Formula/kubekey.rb

```diff
@@ -16,6 +17,9 @@ class Kubekey < Formula
   end
 
   depends_on "go" => :build
+  depends_on "gpgme"
+  # due to missing libbtrfs headers, only supports macos at the moment
```

Comment:
Any issue upstream for this?

Reply:
This is basically the same thing I did with werf

https://github.com/Homebrew/homebrew-core/blob/edbc4965f09e9f82322464d52dd34bde46d50237/Formula/werf.rb#L26-L27

Reply:
We basically just missed the libbtrfs formula for the linux build, there is nothing wrong with the upstream.

Reply:
```suggestion
  # due to missing libbtrfs formula, not needed on macOS
```

Reply:
May be available once we add `btrfs-progs` (#114864)